presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -531,24 +524,13 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 break;
         }
 
-        if (featuresConfig.isCharPadSpaces()) {
-            builder.function(CHAR_EQUAL_PAD_SPACES)
-                    .function(CHAR_NOT_EQUAL_PAD_SPACES)
-                    .function(CHAR_LESS_THAN_PAD_SPACES)
-                    .function(CHAR_LESS_THAN_OR_EQUAL_PAD_SPACES)
-                    .function(CHAR_GREATER_THAN_PAD_SPACES)
-                    .function(CHAR_GREATER_THAN_OR_EQUAL_PAD_SPACES)
-                    .function(CHAR_BETWEEN_PAD_SPACES);
-        }
-        else {
-            builder.function(CHAR_EQUAL_NO_PAD)
-                    .function(CHAR_NOT_EQUAL_NO_PAD)
-                    .function(CHAR_LESS_THAN_NO_PAD)
-                    .function(CHAR_LESS_THAN_OR_EQUAL_NO_PAD)
-                    .function(CHAR_GREATER_THAN_NO_PAD)
-                    .function(CHAR_GREATER_THAN_OR_EQUAL_NO_PAD)
-                    .function(CHAR_BETWEEN_NO_PAD);
-        }
+        builder.function(CHAR_EQUAL_PAD_SPACES)
```

Comment:
Shouldn't NO PAD version be the used one? I know that char coercions in 148 don't allow for NO PAD version to be implemented correctly in joins. However, in 148 NO PAD was used by default, so not we are changing behavior for our client. Is that OK for client? If OK, then I am fine with this commit.